frontend/src/routes/FacilityBooking/CreateBookingDailyView/index.tsx

```diff
@@ -0,0 +1,170 @@
+import React, { useEffect, useState } from 'react'
+import { useHistory, useParams } from 'react-router-dom'
+import { useDispatch, useSelector } from 'react-redux'
+import styled from 'styled-components'
+import 'antd-mobile/dist/antd-mobile.css'
+import 'antd/dist/antd.css'
+
+import { PATHS } from '../../Routes'
+import {
+  setSelectedBlockTimestamp,
+  setSelectedEndTime,
+  setSelectedStartTime,
+  updateDailyView,
+} from '../../../store/facilityBooking/action'
+import { RootState } from '../../../store/types'
+
+import LoadingSpin from '../../../components/LoadingSpin'
+import BookingSection from '../../../components/FacilityBooking/BookingSection'
+import TopNavBarRevamp from '../../../components/TopNavBarRevamp'
+import DailyViewDatesRow from '../../../components/FacilityBooking/DailyViewDatesRow'
+
+const HEADER_HEIGHT = '70px'
+
+const Background = styled.div`
+  background-color: #fff;
+  height: calc(100vh - ${HEADER_HEIGHT});
+  width: (100vw);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0px 36px;
+  overflow: hidden;
+`
+
+const BookingSectionDiv = styled.div`
+  width: 100vw;
+  height: 100%;
+  overflow-y: scroll;
+
+  scrollbar-width: none;
+  ::-webkit-scrollbar {
+    display: none;
+  }
+`
+
+const TitleText = styled.h2`
+  font-family: Lato;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 22px;
+  margin-top: 0.7rem;
+`
+
+/**
+ * # CreateBookingDailyView
+ * Path: `/facility/booking/dayview/create/:facilityId'`
+ *
+ * ## Page Description
+ * This page is accessed after the user clicks on the `Book Facility` button on the `ViewBookingDailyView` page
+ * The user would select a starting time, followed by an ending time, on the timeline.
+ * Once the selection is done, the user would be automatically redirected to the CreateBooking page
+ *
+ * ## Components
+ * - `LoadingSpin`
+ * - `BookingSection`
+ * - `TopNavBarRevamp`
+ * - `DailyViewDatesRow`
+ *
+ * ## Redux
+ * ### Action functions
+ *
+ * #### /store/facilityBooking/action
+ * - `setSelectedBlockTimestamp<timeStamp: number>`
+ * - `setSelectedEndTime<endTime: number>`
+ * - `setSelectedStartTime<startTime: number>`
+ * - `updateDailyView<date: Date, selectedFacilityId: number>`
+ *
+ * ### `useSelector` variables
+ * #### state.facilityBooking
+ * - `selectedFacilityName`
+ * - `isLoading`
+ * - `selectedStartTime`
+ *
+ * @remarks
+ * <any remarks on this component type in here>
+ * I made this page last AY! - 2022 FE Head
+ *
+ */
```

Comment:
LGTM, but we need to mess with the configurations because the docs are not grouped properly

![image](https://user-images.githubusercontent.com/46131429/190912644-c093bc97-e6f7-4cea-a569-1f2a6e955a60.png)